modules/aioseop_sitemap.php

```diff
@@ -649,8 +649,8 @@ public function add_post_types() {
 			$this->default_options['taxonomies']['default']              = array_keys( $this->default_options['taxonomies']['initial_options'] );
 			$this->default_options['excl_categories']['initial_options'] = $this->get_category_titles();
 
-			$post_name = __( ' Post Type', 'all-in-one-seo-pack' );
-			$tax_name  = __( ' Taxonomy', 'all-in-one-seo-pack' );
+			$post_name = sprintf( ' %1$s', __( 'Post Type', 'all-in-one-seo-pack' ) );
```

Comment:
These are overly complicated and difficult to read. Something like the following should work just as well.

`$post_name = ' ' . __( 'Post Type', 'all-in-one-seo-pack' ) );`